content/en/docs/Installation/build-from-source.md

```diff
@@ -18,7 +18,7 @@ you should open an [issue in the project's GitHub page](https://github.com/navid
 
 If you don't want to wait, you can try to build the binary yourself, with the following steps.
 
-First, you will need to install [Go 1.18](https://golang.org/doc/install) and
```

Comment:
The minimum version should be 1.19 (the one defined in the [go.mod](https://github.com/navidrome/navidrome/blob/master/go.mod#LL3C2-L3C2) file). I try to always support at least the latest two major versions of Go, so it would be 1.19 and 1.20